app/src/main/java/com/sportsworld/learnsharedflow/MainActivity.kt

```diff
@@ -11,7 +11,10 @@ import androidx.compose.runtime.*
 import androidx.compose.foundation.lazy.*
 import androidx.compose.material.*
 import com.sportsworld.sportcomponentlib.DisplayScreenSetup
-
+import com.sportsworld.sportcomponentlib.RefreshButtonLib
+import com.sportsworld.sportcomponentlib.SportFeatureViewModelLib
+import com.sportsworld.sportcomponentlib.RefreshButtonLib
+import com.sportsworld.sportcomponentlib.ui.icons.RefreshIcon
```

Comment:
Breaks here for me?

Reply:
Hopefully now that RefreshIcon has been added it won't break here.